route/path.go

```diff
@@ -0,0 +1,26 @@
+package route
+
+type Path struct {
+	Type       uint8
+	StaticPath *StaticPath
+	BGPPath    *BGPPath
```

Comment:
i don't like this. It doesn't scale if there are new Paths.
The path itself is quite different for the different protocols. So creating an common interface is not possible here.
Using empty interface would involve reflection which is also no option.
I will think about this. Maybe i have an idea